Automattic/wp-calypso
# Reader: rename blacklist to blocklist

#### Changes proposed in this Pull Request

Renames the existing `author-name-blacklist` and `site-description-blacklist` libraries to `author-name-blocklist` and `site-description-blocklist` respectively.

Also moves the associated test suites from Chai to Jest assertions, and addresses a few lint errors.

#### Testing instructions



Visit your Reader stream at http://calypso.localhost:3000/read. Ensure that author names appear as normal on posts.

Visit a blog stream like http://calypso.localhost:3000/read/feeds/63937090, and ensure that the site description appears under the site title in the header.
